src/AccessibilityInsights.Extensions.Telemetry/Extensions.Telemetry.csproj

```diff
@@ -43,6 +43,9 @@
     <Reference Include="Microsoft.ApplicationInsights, Version=2.8.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.ApplicationInsights.2.8.1\lib\net46\Microsoft.ApplicationInsights.dll</HintPath>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
Not sure why this reference was added?

Reply:
Investigating

---
In reply to: [283034705](https://github.com/microsoft/accessibility-insights-windows/pull/353#discussion_r283034705) [](ancestors = 283034705)